install.rb

```diff
@@ -147,14 +147,10 @@ def prepare_installation
   if InstallOptions.configdir
     configdir = InstallOptions.configdir
   elsif is_windows?
-    begin
-      require 'rubygems'
-      require 'win32/dir'
-    rescue LoadError => e
-      puts "Cannot run on Microsoft Windows without the sys-admin, win32-process, win32-dir & win32-service gems: #{e}"
-      exit(-1)
-    end
-    configdir = File.join(Dir::COMMON_APPDATA, "PuppetLabs", "hiera", "etc")
+    path = File.join(File.dirname(__FILE__), "lib", "hiera", "util", "win32")
```

Comment:
I verified that this indeed loads properly, by slightly modifying the code to emit some output as

```ruby
app_data = Hiera::Util::Win32.get_common_appdata()
puts "found app_data: #{app_data}"
configdir = File.join(app_data, "PuppetLabs", "hiera", "etc")
```

Output

```
C:\Users\Administrator\Documents\hiera>ruby install.rb
found app_data: C:\ProgramData
install -c -p -m 0644 ext/hiera.yaml C:\ProgramData/PuppetLabs/hiera/etc/hiera.yaml
install -c -p -m 0755 C:/Users/ADMINI~1/AppData/Local/Temp/hiera-wrapper20170419-4004-d5xlog C:/tools/ruby23/bin/hiera.bat
install -c -p -m 0755 C:/Users/ADMINI~1/AppData/Local/Temp/hiera-binfile20170419-4004-7nbazk C:/tools/ruby23/bin/hiera
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/backend
install -c -p -m 0644 lib/hiera/backend/json_backend.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/backend/json_backend.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/backend
install -c -p -m 0644 lib/hiera/backend/yaml_backend.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/backend/yaml_backend.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/backend.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/backend.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/config.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/config.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/console_logger.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/console_logger.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/error.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/error.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/fallback_logger.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/fallback_logger.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/filecache.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/filecache.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/interpolate.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/interpolate.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/noop_logger.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/noop_logger.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/puppet_logger.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/puppet_logger.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/recursive_guard.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/recursive_guard.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/util
install -c -p -m 0644 lib/hiera/util/win32.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/util/win32.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/util.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/util.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera
install -c -p -m 0644 lib/hiera/version.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera/version.rb
mkdir -p -m 755 C:/tools/ruby23/lib/ruby/site_ruby/2.3.0
install -c -p -m 0644 lib/hiera.rb C:/tools/ruby23/lib/ruby/site_ruby/2.3.0/hiera.rb
```